doc/design/data-retention-policy.md

```diff
@@ -0,0 +1,58 @@
+# Data Retention Policy
+
+Dandihub data storage on AWS EFS is expensive, and we suppose that significant portions of the data
+currently stored are no longer used. Data migration is where the cost becomes extreme.
+
+## Persistent Data locations
+
+Each user has access to 2 locations: `/home/{user}` and `/shared/`.
+
+Within jupyterhub `/home/{user}` the user always sees `/home/jovyan`, but is stored in EFS as their GitHub
+username.
```

Comment:
We were previously considering providing a `/scratch` directory for each user that is automatically cleaned up after 30 days.  In addition to the policy for the `/home/jovyan` directory, do we also want to implement a `/scratch` directory with a 30 day clean up policy?